src/views/Dashboard/Customer/Home/GeneticData/RequestAnalysis/SuccessPage.vue

```diff
@@ -91,82 +85,50 @@ export default {
 
   async mounted() {
     await this.getAnalysisOrderDetail()
-    await this.getServiceDetail()
-    await this.getGeneticData()
-    await this.getAnalysisStatus()
 
+    if (this.orderStatus === "Cancelled") {
+      this.stepperItems[3].title = "Cancelled"
+    }
+
+    if (this.orderStatus === "Paid") {
+      this.getAnalysisStatus()
+    }
   },
 
   methods: {
     async getAnalysisOrderDetail() {
       const analysisOrderId = this.$route.params.id
       const analysisOrderDetail = await queryGeneticAnalysisOrders(this.api, analysisOrderId)
-      this.geneticOrderAnalysisDetail = analysisOrderDetail
-      this.orderStatus = this.geneticOrderAnalysisDetail.status
-      this.trackingId = this.geneticOrderAnalysisDetail.geneticAnalysisTrackingId
+
+      if (analysisOrderDetail) {
+        this.orderStatus = analysisOrderDetail.status
+        this.geneticOrderAnalysisDetail = analysisOrderDetail
+        this.trackingId = this.geneticOrderAnalysisDetail.geneticAnalysisTrackingId
+      }
     },
 
     async getAnalysisStatus() {
-      const details = await queryGeneticAnalysisStorage(this.api, this.trackingId)
-      if (details.status === "InProgress") {
-        this.isProcessing = true
-      }
+      this.isRegistered = false
+      this.isRejected = false
+      this.isInProgress = false
 
-      if (this.orderStatus === "Cancelled") {
-        this.stepperItems[3].title = "Cancelled"
-        this.isCancelled = true
-      }
+      this.geneticAnalysis = await queryGeneticAnalysisStorage(this.api, this.trackingId)
 
-      if (this.orderStatus === "Failed") {
-        this.stepperItems[3].title = "Rejeted"
+      if (this.geneticAnalysis.status === "Rejected") {
+        this.stepperItems[3].title = "Rejected"
         this.isRejected = true
       }
 
-      this.rejectTitle = details.rejectedTitle
-      this.rejectDesc = details.rejectedDescription
-      
-    },
-
-    async getServiceDetail() {
-      const serviceDetail = await queryGetGeneticAnalystServiceById(this.api, this.geneticOrderAnalysisDetail.serviceId)
-    
-      let {
-        id: serviceId,
-        ownerId: analystId,
-        info: {
-          name: serviceName,
-          pricesByCurrency: priceDetail,
-          expectedDuration: {
-            duration,
-            durationType
-          },
-          description,
-          testResultSample
-        }     
-      } = serviceDetail
-
-      const analystsInfo = await queryGeneticAnalysts(this.api, analystId)
-      const service = {
-        serviceId,
-        analystId,
-        serviceName,
-        priceDetail,
-        duration,
-        durationType,
-        description,
-        testResultSample,
-        analystsInfo
+      if (this.geneticAnalysis.status === "Registere") {
```

Comment:
Typo?

Reply:
yepp sorry , fixed it . thank you @agustinustheo